src/app/components/manuscript-thumbnails-viewer/manuscript-thumbnails-viewer.component.scss

```diff
@@ -0,0 +1,56 @@
+.thumbnails-view{
+    height: 100%;
+    background: #617d8a;
```

Comment:
Add this and every other UI color in the themes map (`assets/scss/_themes.scss`). Follow instructions in README  to know how to handle color themes (Section ### 4.1 - Handle color theme).